CMakeLists.txt

```diff
@@ -96,6 +96,17 @@ if (WITH_MPFR)
     set(HAVE_SYMENGINE_MPFR yes)
 endif()
 
+# Piranha
+set(WITH_PIRANHA no
+    CACHE BOOL "Build with Piranha")
+
+if (WITH_PIRANHA)
+    find_package(Piranha REQUIRED)
+    include_directories(${PIRANHA_INCLUDE_DIRS})
+    set(LIBS ${LIBS} pthread)
```

Comment:
It will not support our COMMON_DIR though.

Sent from my mobile phone.
On Jun 6, 2015 8:44 AM, "Isuru Fernando" <notifications@github.com> wrote:

> In CMakeLists.txt
> <https://github.com/sympy/symengine/pull/464#discussion_r31868568>:
>
> > @@ -96,6 +96,17 @@ if (WITH_MPFR)
> >      set(HAVE_SYMENGINE_MPFR yes)
> >  endif()
> >
> > +# Piranha
> > +set(WITH_PIRANHA no
> > +    CACHE BOOL "Build with Piranha")
> > +
> > +if (WITH_PIRANHA)
> > +    find_package(Piranha REQUIRED)
> > +    include_directories(${PIRANHA_INCLUDE_DIRS})
> > +    set(LIBS ${LIBS} pthread)
>
> @certik <https://github.com/certik>, CMake has FindBoost.cmake as well.
> (It's there in v2.8)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/symengine/pull/464/files#r31868568>.
>